docs/examples/perf_analyzer/decoding_pipeline.py

```diff
@@ -0,0 +1,46 @@
+# The MIT License (MIT)
+#
+# Copyright (c) 2021 NVIDIA CORPORATION
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy of
+# this software and associated documentation files (the "Software"), to deal in
+# the Software without restriction, including without limitation the rights to
+# use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of
+# the Software, and to permit persons to whom the Software is furnished to do so,
+# subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS
+# FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR
+# COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER
+# IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN
+# CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+import nvidia.dali as dali
+import nvidia.dali.types as types
+
+
+def _parse_args():
+    import argparse
+    parser = argparse.ArgumentParser(description="Serialize pipeline and save it to file")
+    parser.add_argument('file_path', type=str, help='Path, where to save serialized pipeline')
+    return parser.parse_args()
+
+
+@dali.pipeline_def(batch_size=256, num_threads=8, device_id=0)
```

Comment:
```suggestion
@dali.pipeline_def(batch_size=256, num_threads=4, device_id=0)
```
I think 8 is too much (it would consume a lot of GPU memory).